src/Login/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import "./Login.css";
+import {toast, ToastContainer} from "react-toastify";
+import LoginForm from "./LoginForm";
+import "../common/Notify.css"
+
+function Login() {
+
+    const url = window.location.href;
+    const urlParts = url.split('?');
+    const isSuccessSignUp : boolean = urlParts[1] === "successfulSignUp";
+    const isSuccessDelete : boolean = urlParts[1] === "successfulDelete";
+
+    const notifySignUp = () => toast.success('Se ha registrado exitosamente!', {
+        position: "top-center",
+        autoClose: 7000,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined});
+
+    const notifyDelete = () => toast.info('Lamentamos que te hayas ido… Eperamos verte pronto nuevamente!', {
+        position: "top-center",
+        autoClose: 7000,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined});
+
+    if(isSuccessSignUp){
+        notifySignUp();
+        window.history.replaceState("","",urlParts[0])
+    }
+
+    if(isSuccessDelete){
+        notifyDelete();
+        window.history.replaceState("","",urlParts[0])
+    }
+
+    return (
+        <div>
```

Comment:
Always try to assign a custom classname to your screens, so on the css you can select that specific screen and all the changes you made are applied only to your screen.

In this case your main `<div/> ` should be something like this:
`<div className={"login-screen"} />`